### PR TITLE
Pull in new datasources and upgrade dependencies

### PR DESCRIPTION
**Pull request by ChrisStrykesAgain**
The goal here was to pull in the relevant datasources being developed for SLE so that they can be used with SLS. 

Doing so involved several modifications to this existing SLS Grafana repo, including upgrading the Grafana version used, updating the dependencies of the existing notebook and test monitor plugins, and changing a function in the notebook plugin to remove a dependency on a private package that prevented it from building cleanly.

Specific plugins added included the Workspace, Tag, and Systems datasources. The Plotly plugin from the SLE repo was also pulled in, in lieu of the one pre-existing in this repo.

### CODE REVIEW

**Comment by ChrisStrykesAgain** [comment]
FYI, I decided to convert this to a draft while my team does some interactive testing with a local build. I don't foresee any further changes, but decided it was probably for the best to get the extra testing.

**Review by mure — APPROVED** [review]
Thanks for figuring all of this out. 
One thought - if we're pulling the plotly panel from the SLE repo, let's go ahead and delete it from this one.

File: build.sh
Comment (cameronwaterman) [inline build.sh:35]: Not a big deal here but we generally suggest to indent using spaces.

File: config/datasources.yml
Comment (cameronwaterman) [inline config/datasources.yml:6]: Probably worth verifying, but I think in our testing adding a uid to an existing data source that has been provisioned and used in dashboards caused Grafana to see it as a new data source (since it generates a uid otherwise), and breaks dashboards.

**Review by cameronwaterman — APPROVED** [review]

**Review by pvallone — APPROVED** [review]

File: config/datasources.yml
Comment (ChrisStrykesAgain) [reply config/datasources.yml:6]: That doesn't seem to be the case in the interactive testing we've done, but I also don't claim a fully tested upgrade scenario. That workflow is blocked by at least this issue: #50  

I'd prefer not to pull that effort in to this PR since the issue is technically pre-existing in my mind;

File: build.sh
Comment (ChrisStrykesAgain) [reply build.sh:35]: Agreed and fixed.

File: config/datasources.yml
Comment (cameronwaterman) [reply config/datasources.yml:6]: Sure I wouldn't expect that fix to be included in this PR. But I would at least ensure that adding the uid to existing data sources doesn't break dashboards that reference them, because that would likely be a major issue for customers.

Comment (ChrisStrykesAgain) [reply config/datasources.yml]: I fixed this.

**Comment by ChrisStrykesAgain** [comment]
Not sure how I ended up with a conflict here, but could someone with write access give me a hand? We're in a good spot to merge based on testing.

**Comment by cameronwaterman** [comment]
> Not sure how I ended up with a conflict here, but could someone with write access give me a hand? We're in a good spot to merge based on testing.

Is the conflict not something you can resolve? I agree it is odd that you somehow got a conflict

**Comment by mure** [comment]
@ChrisStrykesAgain You somehow have conflicts with a 2-year old commit. Since this is a fork PR, I'm not sure if I have the permissions to fix it on your end.

**Comment by ChrisStrykesAgain** [comment]
I actually figured it out. There was a message that said something to the effect that I didn't have write permissions on the origin, so I should ask someone who does. I didn't think about the fact I could just fix it in my branch instead.

**Pull request merged**
